### DISCUSSION TITLE
Anleitung Installation

### DISCUSSION THREAD

**Discussion by Slin-nilS** [diefenbecker/opencistern#2] [Ideas]
Servus, 

könnt man die Anleitung entsprechend in die Read Me verlinken?

[http://blog.bubux.de/fuellstandmessung-der-zisterne-mittels-esp8266-nodemcu-und-ultraschallsensor/](url)

Ich denke das würde bestimmt auch anderen Einsteigern weiterhelfen. Was meint ihr?

VG Nils

**Comment by diefenbecker** [comment]
Guten Morgen,

ja, gute Idee. Kann ich gerne machen.

Gruß
Chris
